### PR TITLE
refactor: expose more column while cli list/get

### PR DESCRIPTION
**Pull request by aiwantaozi**
**Problem:**
variables, settings and resource outputs list did not show the value

**Solution:**
support more columns

**Related Issue:**
#1582

### CODE REVIEW

File: pkg/dao/types/resource_revision.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ import (
 
 type OutputValue struct {
 	Name      string          `json:"name,omitempty"`
-	Value     property.Value  `json:"value,omitempty"`
+	Value     property.Value  `json:"value,omitempty,cli-table-column"`
 	Type      cty.Type        `json:"type,omitempty"`
 	Sensitive bool            `json:"sensitive,omitempty"`
```
Comment (shanewxy) [inline pkg/dao/types/resource_revision.go:14]: Do we need to expose sensitive as well like in variable?

Comment (aiwantaozi) [reply pkg/dao/types/resource_revision.go:14]: Now the outputs API doesn't return the sensitive field, it uses <sensitive> to replace the value. So we customize the variable list, and replace value with <sensitive> for sensitive value.
Variable list:
image

Output list:
image

**Review by shanewxy — APPROVED** [review]
LGTM

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**